$NetBSD: patch-scripts_Makefile.am,v 1.1 2021/05/13 10:47:49 schmonz Exp $

Work around build failure with libc++ >=7.0 on case-insensitive
filesystems (issue #1051).

--- scripts/Makefile.am.orig	2017-07-22 21:14:53.000000000 +0000
+++ scripts/Makefile.am
@@ -4,6 +4,6 @@ if BUILD_CLIENT
 endif
 CLEANFILES = $(bin_SCRIPTS)
 
-mosh:	mosh.pl ../VERSION Makefile
-	@sed -e "s/\@VERSION\@/`cat ../VERSION`/" -e "s/\@PACKAGE_STRING\@/@PACKAGE_STRING@/" $(srcdir)/mosh.pl > mosh
+mosh:	mosh.pl ../case-insensitive-VERSION Makefile
+	@sed -e "s/\@VERSION\@/`cat ../case-insensitive-VERSION`/" -e "s/\@PACKAGE_STRING\@/@PACKAGE_STRING@/" $(srcdir)/mosh.pl > mosh
 	@chmod a+x mosh
